.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 node_modules/
 coverage/
+package-lock.json
```

Comment:
Do not ignore this file. It should always be committed

Reply:
@adesege worked on feedback